DevEdu.API/DevEdu.Business/ValidationHelpers/TopicValidationHelper .cs

```diff
@@ -19,5 +22,14 @@ public void CheckTopicExistence(int topicId)
             if (topic == default)
                 throw new EntityNotFoundException(string.Format(ServiceMessages.EntityNotFoundMessage, nameof(topic), topicId));
         }
+        public void CheckTopicsExistence(List<CourseTopicDto> topics)
+        {
+            var topicsFromBd = _topicRepository.GetAllTopics();
+            var exsitedTopicIds = topics.Select(s => s.Topic.Id).Where(d => !topicsFromBd.Select(g => g.Id).Contains(d));
```

Comment:
I'm a bit confused with this line of code 
could you rewrite it?..
maybe with usage of Any or All methods